CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 Changes
 =======
+## 1.2.0 / 2020-07-23
```

Comment:
The formatting is a little different than the other entries. I'm not sure if it's best to start using the automated format or our old one

Reply:
Let's keep the current one for now, good catch

Reply:
Let's keep the current one for now, good catch